opencensus/stats/measurement.py

```diff
@@ -0,0 +1,17 @@
+from opencensus.stats import measure
+
+class Measurement(object):
+    def __init__(self, measure, value):
+        self.measure = measure.Measure(measure)
+        self.value = value
+
+    def get_measure(self):
+        return self.measure.Measure
+
+class MeasurementInt(Measurement):
+    def __init__(self, measure, value):
+        super().__init__(measure, value)
+
+class MeasurementFloat(Measurement):
+    def __init__(self, measure, value):
+        super().__init__(measure, value)
```

Comment:
Add new line at end of file and same for the other files.

Reply:
Ok!